docs/docs/get_started/quickstart.mdx

```diff
@@ -83,7 +83,6 @@ LangChain provides several objects to easily distinguish between different roles
 If none of those roles sound right, there is also a `ChatMessage` class where you can specify the role manually.
 For more information on how to use these different messages most effectively, see our prompting guide.
 
-LangChain provides a standard interface for both, but it's useful to understand this difference in order to construct prompts for a given language model.
```

Comment:
can we just reword, something like:
"LangChain provides a standard interface for both"
-> 
"Langchain provides a common interface that's shared by both LLMs and ChatModels. However it's useful to understand..."

Reply:
@baskaryan I really like your second suggestion "Langchain provides a common interface that's shared by both LLMs and ChatModels. However it's useful to understand..."